docs/admin/daemons.md

```diff
@@ -91,6 +88,14 @@ when the pod is created, so it is ignored by the scheduler).  Therefore:
    by the DaemonSet controller.
  - DaemonSet controller can make pods even when the scheduler has not been started, which can help cluster
    bootstrap.
+   
+Daemon pods do respect [taints and tolerations](/docs/user-guide/node-selection/index.md), but they are
```

Comment:
How about use `/docs/user-guide/node-selection/index.md#taints-and-tolerations-beta-feature`